### PR TITLE
Cleanup backplane script in openAPI definition

### PR DESCRIPTION
**Pull request by Dee-6777**
This is for [OSD-17474](https://issues.redhat.com/browse/OSD-17474)

Changes done :
- cleaned up /backplane/script definition in openAPI yaml file
- generated new-client-pkg

### CODE REVIEW

File: openapi/openapi.yaml [hunk 1]
```diff
@@ -179,37 +179,6 @@ paths:
...
-        200:
-          description: Successful
-          content:
-            application/json:
-              schema:
-                $ref: '#/components/schemas/Scripts'
-
   /backplane/script/{clusterId}:
     get:
       operationId: getScriptsByCluster
```
Comment (samanthajayasinghe) [inline openapi/openapi.yaml:184]: You may have to define the following section for `/backplane/script/{clusterId}:` end point 
```
      summary: 
      parameters:
      responses:
```

Comment (Dee-6777) [reply openapi/openapi.yaml:184]: Hi Sam, it's present in the `/backplane/script/{clusterId}` definition.

**Comment by feichashao** [comment]
CLI is not calling the script endpoint directly, we are safe to go.
https://github.com/openshift/backplane-cli/blob/691eb6f0d3d4cd8a4e62a3494e1f1eed43302ca3/cmd/ocm-backplane/script/listScripts.go#L84
https://github.com/openshift/backplane-cli/blob/691eb6f0d3d4cd8a4e62a3494e1f1eed43302ca3/cmd/ocm-backplane/script/describeScript.go#L84

**Pull request merged**